File: python/Animal.py
# DESAFIO
# Neste problema, você deverá ler 3 palavras que definem o tipo de animal possível segundo o esquema abaixo, da esquerda para a direita.  Em seguida conclua qual dos animais seguintes foi escolhido, através das três palavras fornecidas.

# Entrada
# A entrada contém 3 palavras, uma em cada linha, necessárias para identificar o animal segundo a figura acima, com todas as letras minúsculas.

# Saída
# Imprima o nome do animal correspondente à entrada fornecida.

A = input()
B = input()
C = input()

if A == 'vertebrado' and B == 'ave' and C == 'carnivoro':
  resposta = 'aguia'

if A == 'vertebrado' and B == 'ave' and C == 'onivoro':
  resposta = 'pomba'

if A == 'vertebrado' and B == 'mamifero' and C == 'onivoro':
  resposta = 'homem'

if A == 'vertebrado' and B == 'mamifero' and C == 'herbivoro':
  resposta = 'vaca'

if A == 'invertebrado' and B == 'inseto' and C == 'hematofago':
  resposta = 'pulga'

if A == 'invertebrado' and B == 'inseto' and C == 'herbivoro':
  resposta = 'lagarta'

if A == 'invertebrado' and B == 'anelideo' and C == 'hematofago':
  resposta = 'sanguessuga'

if A == 'invertebrado' and B == 'anelideo' and C == 'onivoro':
  resposta = 'minhoca'

print(resposta)

File: js/Animal.js
// Desafio

// Neste problema, você deverá ler 3 palavras que definem o tipo de animal possível segundo o esquema abaixo, da esquerda para a direita.  Em seguida conclua qual dos animais seguintes foi escolhido, através das três palavras fornecidas.

// Entrada
// A entrada contém 3 palavras, uma em cada linha, necessárias para identificar o animal segundo a figura acima, com todas as letras minúsculas.

// Saída
// Imprima o nome do animal correspondente à entrada fornecida.

// a função gets é implementada dentro do sistema para ler as entradas(inputs) dos dados e a função print para imprimir a saída(output) de dados.
// Abaixo segue um exemplo de código que você pode ou não utilizar

let x = gets();
let y = gets();
let z = gets();

if ((x == "vertebrado") && (y == "ave") && (z == "carnivoro")) {
    print("aguia\n");
}

if ((x == "vertebrado") && (y == "ave") && (z == "onivoro")) {
    print("pomba");
}

if ((x == "vertebrado") && (y == "mamifero") && (z == "onivoro")) {
    print("homem");
}

if ((x == "vertebrado") && (y == "mamifero") && (z == "herbivoro")) {
    print("vaca");
}

if ((x == "invertebrado") && (y == "inseto") && (z == "hematofago")) {
    print("pulga");
}

if ((x == "invertebrado") && (y == "inseto") && (z == "herbivoro")) {
    print("lagarta");
}

if ((x == "invertebrado") && (y == "anelideo") && (z == "hematofago")) {
    print("sanguessuga");
}

if ((x == "invertebrado") && (y == "anelideo") && (z == "onivoro")) {
    print("minhoca");
}